lib/classifier-reborn/backends/bayes_memory_backend.rb

```diff
@@ -0,0 +1,72 @@
+# Author: Sawood Alam <@ibnesayeed>
```

Comment:
That will be gone. I usually don't add such things, but I have seen many other files here that have Author, License, and some other metadata associated in comments, so I thought it was the style of this code base.